packages/catalog-process/test/createDescriptor.test.ts

```diff
@@ -119,7 +132,23 @@ describe("create descriptor", async () => {
     });
   });
 
+  // To do: this is the only use case that will remain active
```

Comment:
which is the action point for this todo?

Reply:
It's related to [this](https://github.com/pagopa/interop-be-monorepo/pull/739/files#diff-1b1dfb20182a7aa8f7b501fd921b7e02de99664304ce284c19d28fa0edeaa7ebR48) other comment.

Reply:
ok but is this specific todo required?

Reply:
Removed